certbot/src/certbot/crypto_util.py

```diff
@@ -386,7 +415,7 @@ def verify_fullchain(renewable_cert: interfaces.RenewableCert) -> None:
 
 
 def get_sans_from_cert(
-    cert: bytes, typ: Union[acme_crypto_util.Format, int] = acme_crypto_util.Format.PEM
+    cert: bytes, typ: Optional[Union[acme_crypto_util.Format, int]] = None
```

Comment:
I personally have a strong preference for union types over `Optional` since it doesn't require imports, and now that 3.9 is dropped we can use [union types.](https://docs.python.org/3/library/stdtypes.html#types-union) Technically there's no deprecation or preference PEP that I've seen, but just seems cleaner imo.
```suggestion
    cert: bytes, typ: Union[acme_crypto_util.Format, int] | None = None
```

or

```suggestion
    cert: bytes, typ:acme_crypto_util.Format | int | None = None
```